raiden-dapp/tests/e2e/specs/scenario.spec.ts

```diff
@@ -96,7 +97,8 @@ describe('dApp e2e tests', () => {
     navigateToUDC();
     enterUDCWithdrawalAmount(uiTimeout);
     withdrawUDCTokens();
-    navigateBackToAccountMenu();
+    reloadWholeApplication();
```

Comment:
Maybe we could add a logging here to see when this happens timewise in relation to the server.

Reply:
Atm I don't know how to get a relative time value here....

Reply:
Not that important.